sys/sys/link_elf.h

```diff
@@ -110,6 +110,12 @@ int _rtld_addr_phdr(const void *, struct dl_phdr_info *);
 int _rtld_get_stack_prot(void);
 int _rtld_is_dlopened(void *);
 
+struct dl_c18n_compart_state;
```

Comment:
Including the header should give this a complete type (presumably from some machine/foo.h) rather than a forward declaration

Reply:
Ack